### PR TITLE
Fix for 73

### PR DESCRIPTION
**Pull request by sbrossie**

### CODE REVIEW

File: queue/src/main/java/org/killbill/queue/DefaultQueueLifecycle.java [hunk 1]
```diff
@@ -288,4 +300,22 @@ private void sleepSporadically(final long loopTimeMsec) throws InterruptedExcept
...
+
+    private interface RunnableRawCallback {
+        void callback() throws InterruptedException;
+    }
+
+    private void withHandlingRuntimeException(final RunnableRawCallback cb) throws InterruptedException {
+        try {
+            cb.callback();
+        } catch (final DBIException e) {
+            log.warn(String.format("%s: Thread %s got DBIException exception: %s",
```
Comment (pierre) [inline queue/src/main/java/org/killbill/queue/DefaultQueueLifecycle.java]: image

https://www.slf4j.org/faq.html#logging_performance 😏

Comment (sbrossie) [reply queue/src/main/java/org/killbill/queue/DefaultQueueLifecycle.java]: Ok, actually, now i remember.. most logs in the `queue` code are written this way:
* Should i only update this new one ?
* Should i  update the whole file?
* Should i  update the whole queue code base?
* Should i  update nothing ?

Comment (pierre) [reply queue/src/main/java/org/killbill/queue/DefaultQueueLifecycle.java]: Up to you. FWIW my vote would be to update the current file. 👼 

> Leave Things BETTER than you found them. ~ Robert Baden Powell

Comment (sbrossie) [reply queue/src/main/java/org/killbill/queue/DefaultQueueLifecycle.java]: > FWIW my vote would be to update the current file. 👼

👍 

> Robert Baden Powell 

Really, who is that Dude anyways :-)

Comment (sbrossie) [reply queue/src/main/java/org/killbill/queue/DefaultQueueLifecycle.java]: @pierre see cb6d28806243f280ee6

**Pull request merged**